streams/src/test/java/org/apache/kafka/streams/integration/InternalTopicIntegrationTest.java

```diff
@@ -83,24 +86,56 @@ public void before() {
         streamsConfiguration = new Properties();
         streamsConfiguration.put(StreamsConfig.APPLICATION_ID_CONFIG, applicationId);
         streamsConfiguration.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, CLUSTER.bootstrapServers());
-        streamsConfiguration.put(StreamsConfig.ZOOKEEPER_CONNECT_CONFIG, CLUSTER.zKConnectString());
         streamsConfiguration.put(StreamsConfig.KEY_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
         streamsConfiguration.put(StreamsConfig.VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
         streamsConfiguration.put(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getPath());
         streamsConfiguration.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
     }
 
+    //@Test
+    public void testCreateTopic() throws Exception {
+
+        StreamsConfig streamsConfig = new StreamsConfig(streamsConfiguration);
+        StreamsKafkaClient streamsKafkaClient = new StreamsKafkaClient(streamsConfig);
+
+        String topicName = "testTopic" + mockTime.milliseconds();
+        InternalTopicConfig topicConfig = new InternalTopicConfig(topicName,
+                Collections.singleton(InternalTopicConfig.CleanupPolicy.compact),
+                Collections.<String, String>emptyMap());
+
+        streamsKafkaClient.createTopic(topicConfig, 1, 1, 0);
+
+        assertTrue(streamsKafkaClient.topicExists(topicName));
+
+    }
+
+    //@Test
+    public void testDeleteTopic() throws Exception {
```

Comment:
Do we need this? I don't think we ever delete topics? Also the test is commented out